### PR TITLE
add fast boot support in syncd

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: syncd/syncd.cpp [hunk 1]
```diff
@@ -993,9 +993,26 @@ cmdOptions handleCmdLine(int argc, char **argv)
...
+                    options.startType = 0;
+                }
+                else if (std::string(optarg) == "warm")
+                {
+                    options.startType = 1;
+                }
+                else if (std::string(optarg) == "fast")
+                {
+                    options.startType = 2;
+                SWSS_LOG_NOTICE("start type: %d", options.startType);
```
Comment (stcheng) [inline syncd/syncd.cpp]: remove this line

**Comment by stcheng** [comment]
consider later add enums in common repo

**Comment by lguohan** [comment]
added sai boot type defines

**Comment by stcheng** [comment]
ACK

**Pull request merged**